tests/antithesis/test-template/Dockerfile

```
@@ -8,3 +8,4 @@ COPY . .
 WORKDIR /build/tests
 RUN go build -o /opt/antithesis/entrypoint/entrypoint -race ./antithesis/test-template/entrypoint/main.go
 RUN go build -o /opt/antithesis/test/v1/robustness/singleton_driver_main -race ./antithesis/test-template/robustness/main.go
+RUN go build -o /opt/antithesis/test/v1/robustness/finally_driver_main -race ./antithesis/test-template/robustness/finally/main.go
```

```suggestion
RUN go build -o /opt/antithesis/test/v1/robustness/finally_validate -race ./antithesis/test-template/robustness/finally/main.go
```
Name pattern is `<test_name>/finally_<command>` 
 http://antithesis.com/docs/test_templates/test_composer_reference/?sid=8c9e8.7&sterm=driver#finally-command